proselint/registry/__init__.py

```diff
@@ -56,17 +56,15 @@ def get_all_enabled(
         """Filter registered checks by config values based on their keys."""
```

Comment:
The expectation that checks should be sorted by specificity should be documented here.